release_docs/RELEASE.txt

```diff
@@ -89,6 +89,17 @@ New Features
 
     Library:
     --------
+    - Fix for CVE-2019-8396
+
+      Malformed HDF5 files may have truncated content which does not match
+      the expected size. When H5O__pline_decode() attempts to decode these it
+      may read past the end of the allocated space leading to heap overflows
+      as bounds checking is incomplete.
+
+      The fix ensures each element is within bounds before reading.
+
+      (2022/11/09 - HDFFV-10712, CVE-2019-8396, GitHub #2209)
+
```

Comment:
I'm sorry I didn't notice this before.  This entry should be in the Bug Fixes section, not the New Features section.